Test Pipelines are unable to fetch Keycloak-admin-UI from Sonatype Repo

### Describe the bug

Keycloakk QE test pipelines use keycloak- version with current timestamp to differentiate between different builds but the current upstream repo fetches the keycloak-admin-ui from NIghhtly releases with a version 999-Snapshot which causes the failure.

### Version

19.0.0

### Expected behavior

Keycloak should be buit successfuly in the test pipelines

### Actual behavior

Following errors occurs in the test pipelines:
`[ERROR] Failed to execute goal on project integration-arquillian-servers-auth-server-undertow: Could not resolve dependencies for project org.keycloak.testsuite:integration-arquillian-servers-auth-server-undertow:jar:999-SNAPSHOT-pr-13393-20220728-124402: Could not find artifact org.keycloak:keycloak-admin-ui:jar:999-SNAPSHOT -> [Help 1]`

### How to Reproduce?

run the command :

mvn clean install -Pdistribution

### Anything else?

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -67,6 +67,8 @@
         <jboss.as.version>7.2.0.Final</jboss.as.version>
         <jboss.as.subsystem.test.version>7.5.22.Final-redhat-1</jboss.as.subsystem.test.version>
 
+        <keycloak.admin-ui.version>${project.version}</keycloak.admin-ui.version>
+
         <!-- Versions used mostly for Undertow server, aligned with WildFly -->
         <jboss.aesh.version>0.66.19</jboss.aesh.version>
         <aesh.version>2.4</aesh.version>
@@ -1418,7 +1420,7 @@
             <dependency>
                 <groupId>org.keycloak</groupId>
                 <artifactId>keycloak-admin-ui</artifactId>
-                <version>${project.version}</version>
+                <version>${keycloak.admin-ui.version}</version>
             </dependency>
 
             <!-- Openshift -->

--- a/testsuite/integration-arquillian/pom.xml
+++ b/testsuite/integration-arquillian/pom.xml
@@ -206,6 +206,24 @@
                 <artifactId>undertow-embedded</artifactId>
                 <version>${undertow-embedded.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.keycloak</groupId>
+                <artifactId>keycloak-dependencies-admin-ui-wrapper</artifactId>
+                <type>pom</type>
+                <version>${project.version}</version>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.keycloak</groupId>
+                <artifactId>keycloak-admin-ui</artifactId>
+                <version>${keycloak.admin-ui.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
         </dependencies>
     </dependencyManagement>
     
@@ -574,5 +592,20 @@
             </properties>
         </profile>
     </profiles>
+
+    <repositories>
+        <repository>
+            <id>sonatype-snapshots</id>
+            <name>Sonatype Snapshots</name>
+            <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+            </snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+    </repositories>
 
 </project>

